superset/exceptions.py

```diff
@@ -52,35 +52,57 @@ class SpatialException(SupersetException):
     pass
 
 
-class DatabaseNotFound(SupersetException):
+class CsvException(SupersetException):
+    def __init__(self, msg, orig_e):
+        super(CsvException, self).__init__(msg)
+        self.orig = orig_e
+
+
+class ConversionException(CsvException):
+    status = 400
+
+
+class DatabaseAlreadyExist(CsvException):
+    status = 400
+
+
+class DatabaseNotFound(CsvException):
     status = 400
 
 
-class DatabaseCreationException(SupersetException):
+class DatabaseCreationException(CsvException):
     status = 400
 
 
-class DatabaseDeletionException(SupersetException):
+class DatabaseDeletionException(CsvException):
+    status = 500
+
+
+class FileSaveException(CsvException):
     status = 500
 
 
-class SchemaNotAllowedCsvUpload(SupersetException):
+class NameNotAllowed(CsvException):
+    status = 400
+
+
+class NoPasswordSuppliedException(CsvException):
     status = 400
 
 
-class NameNotAllowed(SupersetException):
+class NoUsernameSuppliedException(CsvException):
     status = 400
 
 
-class TableCreationException(SupersetException):
+class SchemaException(CsvException):
     status = 400
 
 
-class NoPasswordSuppliedException(SupersetException):
+class SchemaNotAllowedCsvUpload(CsvException):
```

Comment:
What's the difference here? isch schema-exception for when a schema is supplied but it does not exist? maybe be more precise?